db/migrate/20160818091421_add_missing_view_permissions2.rb

```diff
@@ -0,0 +1,21 @@
+class AddMissingViewPermissions2 < ActiveRecord::Migration
```

Comment:
Nitpick: please notice this now adds more then view permissions which is good because I had missing permissions in my discovery role but maybe change the name?